src/application/room/room.util.ts

```diff
@@ -0,0 +1,22 @@
+import { RoomModel } from '../../infrastructure/data-access/typeorm/room.entity';
+
+export class RoomUtil {
+  static getRoomModel(roomData: any): RoomModel {
+    const roomModel = new RoomModel();
+    roomModel.roomId = roomData.room_id || roomData.roomId || null;
```

Comment:
cách code này e nghĩ hơi dài dòng. 
Ví dụ nếu thằng roomModel có 100 cái thông tin thì a phải define ra thành 100 dòng ntn.
Không hợp lí lắm
A có thể chạy vòng for để làm cái này

Reply:
@TuanHai37 Chỗ ni apply vòng lặp kiểu ren em suggest anh xem với

Reply:
for (let item in roomModel) {
     roomModel.item = roomData.item || null
}
return roomModel